### DISCUSSION TITLE
Proxy requests to external services after authentication

### DISCUSSION THREAD

**Discussion by VaZark** [directus/directus#11853] [Feature Requests]
Directus gives its clients a perfect solution for user authentication and management. Sometimes services are distributed across serverless functions or already hosted elsewhere in a different language. 

It will make it easier to migrate incrementally and/or host other services through a central API via directus, if we have an option to proxy requests to other endpoints. It also reduces code repetition by letting directus handle user authentication and related security.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
